nav2_smac_planner/include/nav2_smac_planner/smac_planner_lattice.hpp

```diff
@@ -88,6 +88,30 @@ class SmacPlannerLattice : public nav2_core::GlobalPlanner
     const geometry_msgs::msg::PoseStamped & goal,
     std::function<bool()> cancel_checker) override;
 
+  /**
```

Comment:
Get rid of these 4x getters, they're unused

Reply:
Done